src/ListGroup.js

```diff
@@ -9,6 +9,45 @@ class ListGroup extends React.Component {
       (item, index) => cloneElement(item, { key: item.key ? item.key : index })
     );
 
+    let childrenAnchors = false;
+
+    if (!this.props.children) {
+      return this.renderDiv(items);
+    } else {
+
+      if (Array.isArray(this.props.children)) {
+        this.props.children.forEach((child) => {
+          if (child.props.href) {
+            childrenAnchors = true;
+          }
+        });
```

Comment:
Using [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here would be a little faster since it immediately returns on the first true result. Also slightly cleaner code.